ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AggregatingAttestationPool.java

```diff
@@ -128,39 +126,48 @@ public synchronized void add(final ValidatableAttestation attestation) {
     }
   }
 
+  private Optional<Int2IntMap> getCommitteesSize(final Attestation attestation) {
+    if (attestation.requiresCommitteeBits()) {
+      final AttestationData attestationData = attestation.getData();
+      LOG.debug(
+          "Committees size was not found for target root {} for attestation at slot {}. Will attempt to retrieve it using the relevant state.",
```

Comment:
i think id still delete 'will attempt...'

Reply:
I deleted the whole log and moved it to `getOrCreateAttestationGroup`. It is now:

LOG.debug("Committees size was not available for attestation at slot {} and block root {}. Will NOT add this attestation to the pool.",attestationData.getTarget().getRoot(),attestationData.getBeaconBlockRoot());